include/deal.II/dofs/dof_renumbering.h

```diff
@@ -1273,6 +1273,36 @@ namespace DoFRenumbering
     std::vector<types::global_dof_index> &new_dof_indices,
     const DoFHandler<dim, spacedim>      &dof_handler);
 
+  /**
+   * Reorder the degrees of freedom in lexicographic order of support points.
+   *  The tolerance used to determine whether two support points are equal.
```

Comment:
```suggestion
   * The tolerance is used to determine whether two support points are equal.
```
Do you mean to say "on two adjacent cells"?

Reply:
Do I assume right that you mean "globally lexicographic", as opposed to "lexicographically on each cell"? Perhaps that might also be useful to state.